core/vm/gas_table.go

```diff
@@ -341,14 +343,14 @@ func gasCall(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize
 	var (
 		gas            uint64
 		transfersValue = !stack.Back(2).IsZero()
-		address        = common.Address(stack.Back(1).Bytes20())
-		exist, err     = evm.StateDB.Exist(address)
+		address, err   = common.Bytes20ToAddress(stack.Back(1).Bytes20()).InternalAddress()
+		exist, _       = evm.StateDB.Exist(*address)
```

Comment:
not catching the error ?

Reply:
evm.StateDB.Exist() no longer returns an error